### PR TITLE
Add installation script to download the latest version from S3

### PR DESCRIPTION
**Pull request by muratugureminoglu**
https://github.com/ant-media/Ant-Media-Server/issues/6801

### CODE REVIEW

**Comment by filip-ams** [comment]
Installed/Tested on AWS, webrtc play publish example:

image

**Review by filip-ams — APPROVED** [review]

File: install_ant-media-server.sh [hunk 1]
```diff
@@ -275,7 +275,7 @@ if [ -z "$ANT_MEDIA_SERVER_ZIP_FILE" ]; then
     curl --progress-bar -o ams_community.zip -L "$(curl -s -H "Accept: application/vnd.github+json" https://api.github.com/repos/ant-media/Ant-Media-Server/releases/latest | jq -r '.assets[0].browser_download_url')"   
     ANT_MEDIA_SERVER_ZIP_FILE="ams_community.zip"
   elif [ -n "${LICENSE_KEY}" ]; then
-    check_license=$(curl -s https://api.antmedia.io/?license="${LICENSE_KEY}" | tr -d "\"")
+    check_license=$(curl -s https://api-v2.antmedia.io/?license="${LICENSE_KEY}" | tr -d "\"")
```
Comment (mekya) [inline install_ant-media-server.sh:278]: I need to know where the source code of api-v2.antmedia.io or api.antmedia.io is? 
If they're in a repository, let me know the repo.

If they're not in a repository, please put them into a private repository and write README about how to use and how to deploy.

**Review by mekya — COMMENTED** [review]
Hi @muratugureminoglu ,

I'm ok to merge it. On the other hand, I need to know where the api(-v2).antmedia.io endpoints as mentioned in the comment. 

This is why I'm putting this work as In-Progress

Regards

**Comment by mekya** [comment]
Hi @muratugureminoglu 
It seems there is a problem in CI that blocks me to merge.

FYI

File: install_ant-media-server.sh (hunk 1 above)
Comment (mekya) [reply install_ant-media-server.sh:278]: Hi @muratugureminoglu ,
>If they're not in a repository, please put them into a private repository and write README about how to use and how to deploy.

Thank you for sharing. I want to follow up on my previous message above.

Comment (mekya) [reply install_ant-media-server.sh:278]: Hi @muratugureminoglu ,
> If they're not in a repository, please put them into a private repository and write README about how to use and how to deploy.

I've merged the PR. Thank you. 
I'll keep it in-progress until we clarify the message above.

FYI

Comment (mekya) [reply install_ant-media-server.sh:278]: Thank you @muratugureminoglu,

It seems there are zip files in the repo. It's ok for now. 

Next time, please share the source code and let us have the history of the source code.

**Pull request merged**